examples/default.yaml

```diff
@@ -158,6 +158,10 @@ caCerts:
   #   YOUR-ORGS-TRUSTED-CA-CERT-HERE
   #   -----END CERTIFICATE-----
 
+# Prefix to use for installing guest agent, and containerd with dependencies (if configured)
+# 🟢 Builtin default: /usr/local
+prefix: null
```

Comment:
`prefix` feels a bit too generic to me in the context of `lima.yaml`. Maybe `installPrefix` or even `guestInstallPrefix` would be better?

Reply:
What shout "guestAgentInstallPrefix"

Reply:
It is also used for nerdctl-full install prefix

Reply:
Trying with `guestInstallPrefix`